homeassistant/components/cover/zwave.py

```diff
@@ -110,24 +123,44 @@ class ZwaveGarageDoor(zwave.ZWaveDeviceEntity, CoverDevice):
     def __init__(self, values):
         """Initialize the zwave garage door."""
         ZWaveDeviceEntity.__init__(self, values, DOMAIN)
+        self._cover_state_list = None
+        self._cover_state = None
         self.update_properties()
 
     def update_properties(self):
         """Handle data changes for node values."""
-        self._state = self.values.primary.data
+        self._cover_state = ZWAVE_STATES.get(self.values.primary.data)
```

Comment:
We're going to be adding two new boolean properties where we're comparing against zwave strings, and the cover base class is going to populate the state property based on the boolean properties we're providing here. We shouldn't need to worry about hass strings any more in this file, since we're just providing boolean status now.